R/mail.R

```diff
@@ -0,0 +1,54 @@
+# Kaiaulu - https://github.com/sailuh/kaiaulu
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+
+
+
+#' Creates fake mbox data
+#'
+#' @param mlist specific mailing list associated with this mbox data
+#' @param reply_from email of sender
+#' @param reply_to email of receiver
```

Comment:
same here, would need to do reply_to_author, and reply_to_email (?) and likely to cc too

Reply:
this should be a simple paste0() 1 line of code combining both parameters by the way